### PR TITLE
1.2 feature/admin project details

### PR DESCRIPTION
**Pull request by danbergelt**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by dionys93 — DISMISSED** [review]

File: mission-control-client/src/actions/activeProductActions.js [hunk 1]
```diff
@@ -1,7 +1,39 @@
...
+    axios
+      .all([
+        axiosLabsGraphQL.post("", { query: fullProjectDetailsById(id) }),
+        axiosLabsGraphQL.post("", { query: peopleByProjectId(id) })
+      ])
+      .then(
+        axios.spread((res, res2) => {
+          const project = {
+            project: res.data.data.projects,
+            people: res2.data.data.projectRoles
```
Comment (armandoroman1016) [inline mission-control-client/src/actions/activeProductActions.js:30]: typo?

File: mission-control-client/src/reducers/activeProductStore.js [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+      }
+    case SET_ACTIVE_PROJECT_SUCCESS:
+      return {
+        ...state,
+        project: action.payload
+      }
+    case SET_ACTIVE_PROJECT_FAILURE:
+      return {
+        ...state,
+        err: action.payload
```
Comment (armandoroman1016) [inline mission-control-client/src/reducers/activeProductStore.js]: isLoading should be set to false here?

File: mission-control-client/src/actions/activeProductActions.js (hunk 1 above)
Comment (danbergelt) [reply mission-control-client/src/actions/activeProductActions.js:30]: @armandoroman1016 no it's a special axios thing, next time you're in the zoom I can explain

File: mission-control-client/src/reducers/activeProductStore.js (hunk 2 above)
Comment (danbergelt) [reply mission-control-client/src/reducers/activeProductStore.js]: yep good catch

File: mission-control-client/src/components/dashboard/projects/ProjectList.js [hunk 3]
```diff
@@ -1,88 +1,88 @@
-import React, { useState, useEffect } from "react";
```
Comment (agordon1123) [inline mission-control-client/src/components/dashboard/projects/ProjectList.js:1]: I'll get ProjectList working for my next PR 👍

**Review by agordon1123 — DISMISSED** [review]

**Review by armandoroman1016 — DISMISSED** [review]

**Review by Artyk77-zz — DISMISSED** [review]

**Review by BDesselle — DISMISSED** [review]

**Review by agordon1123 — APPROVED** [review]

**Review by dionys93 — APPROVED** [review]

**Review by BDesselle — APPROVED** [review]

**Pull request merged**